mlrun/feature_store/api.py

```diff
@@ -265,6 +265,12 @@ def get_online_feature_service(
 
     # todo: support remote service (using remote nuclio/mlrun function if run_config)
 
+    for old_name in service.vector.get_feature_aliases().keys():
```

Comment:
what is this?

Reply:
@guy1992l it is not related but it is a bug I fixed  - [fix bug when using "as" if the feature list]